src/common/components/modal/modal.module.scss

```diff
@@ -18,6 +19,9 @@ $modalPadding: 2rem;
     right: 0;
     bottom: 0;
   }
+  @include respond-above(md) {
```

Comment:
i think this is safer
https://getbootstrap.com/docs/4.2/components/modal/#optional-sizes

Reply:
maybe proportional to breakpoint values

Reply:
I set it to 800px, which should be a suitable value. We only have one modal size at the moment (for md and smaller the modals are responsive/fullwidth).

Reply:
👍